scenes/passy_scene_read.c

```diff
@@ -52,6 +52,8 @@ bool passy_scene_read_on_event(void* context, SceneManagerEvent event) {
                     (passy->bytes_total / 1024));
                 popup_set_header(popup, header, 68, 30, AlignLeft, AlignTop);
             }
+        } else if(event.event == PassyCustomEventReaderNoDGXData) {
```

Comment:
So if the card returned file not found, it would error and never show this.  I honestly don't expect you to handle this situation if you don't have a test case for it.  I had mentioned it on discord, but just as a musing, not something we needed to do.